tensorflow-keras/src/main/java/org/tensorflow/keras/optimizers/Adamax.java

```diff
@@ -0,0 +1,283 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the );
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an  BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=======================================================================*/
+package org.tensorflow.keras.optimizers;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import org.tensorflow.Graph;
+import org.tensorflow.Operand;
+import org.tensorflow.Output;
+import static org.tensorflow.keras.optimizers.OptimizerInterface.NAME_KEY;
+import static org.tensorflow.keras.optimizers.OptimizerInterface.assertGraph;
+import org.tensorflow.ndarray.Shape;
+import org.tensorflow.op.Op;
+import org.tensorflow.op.Ops;
+import org.tensorflow.op.Scope;
+import org.tensorflow.op.core.Assign;
+import org.tensorflow.op.core.Constant;
+import org.tensorflow.op.core.Variable;
+import org.tensorflow.op.train.ApplyAdaMax;
+import org.tensorflow.types.TFloat32;
+import org.tensorflow.types.family.TType;
+
+/** Adamax Optimizer that implements the Adamax algorithm. */
+public class Adamax extends org.tensorflow.framework.optimizers.Optimizer
```

Comment:
Can we create a `Adamax` implementation in the framework first and then having its Keras wrapper extend from it like other optimizers? The same applies for all other Keras optimizers that are not present in the framework.

Reply:
Yes.